examples/user/README.md

```diff
@@ -33,7 +33,7 @@
 - [Update Username](UpdateUsername.java)
 
 ## Security Settings Examples
-- [Get Enforced TLS](GetEnforedTLS.java)
+- [Get Enforced TLS](GetEnforcedTLS.java)
```

Comment:
Yes, the file needs to be renamed. Oddly, the class in the file isn't named to match the file name in accordance to general Java naming conventions.